pkgs/by-name/mo/moonlight/package.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  passthru.updateScript = nix-update-script { };
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```

Comment:
What do you consider broken with the `updateScript`? It works as it is on master for me.